modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/CreationMenu.tsx

```diff
@@ -109,36 +110,55 @@ function CreationMenu({
 						primaryItems={primaryItems}
 					/>
 				) : (
-					<ClayButton
-						aria-label={primaryItems[0].label}
-						className="nav-btn nav-btn-monospaced"
+					<LinkOrButton
```

Comment:
@markocikos @dsanz @ethib137 the main conflict will be with this button - my PR only added the conditional displayType prop => `displayType={inEmptyState ? 'primary' : 'secondary'}`

But that's just the low-level collision, Product needs to give clarification I guess